src/sync/sync_user.cpp

```diff
@@ -105,12 +105,14 @@ SyncUser::SyncUser(std::string refresh_token,
                    const std::string identity,
                    const std::string provider_type,
                    std::string access_token,
-                   SyncUser::State state)
+                   SyncUser::State state,
+                   const std::string device_id)
 : m_state(state)
 , m_provider_type(provider_type)
 , m_refresh_token(RealmJWT(std::move(refresh_token)))
 , m_identity(std::move(identity))
 , m_access_token(RealmJWT(std::move(access_token)))
+, m_device_id(std::move(device_id))
```

Comment:
I'm not sure this `move` actually does anything.

Reply:
Move, removed